scrapers/Pornbcn.yml

```
@@ -0,0 +1,34 @@
+name: PornBCN
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pornbcn.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //img[@data-name='image']/@alt
+      Date:
+        selector: //div[@data-date_format]/input[@type='hidden']/@value
+        postProcess:
+          - parseDate: '20060102'
+      Image:
+        selector: //div[contains(@style,'background-image')]/@style
+        postProcess:
+          - replace:
+            - regex: ^.+url\(([^\)]+).+
+              with: $1
+      Studio:
+        Name: //meta[@property='og:site_name']/@content
+      Tags:
+        Name: //meta[@property='article:tag']/@content
+      Performers:
+        Name: //span[@class='namepornstarvi']//a
+      Details: //div[contains(@class,'toggle_content')]/p
```

They sometimes use `<br>` tags to add newlines to their descriptions (see [My First Client](https://pornbcn.com/en/mi-primera-clienta/)) and this wil unfortunately just mash those lines together

I found one scene that uses multiple `<p>` elements as well, so we can account for that as well

```suggestion
      Details:
        selector: //h5[contains(@class, "toggle_title")]/following-sibling::div[not(.//form)]//text()
        concat: "\n"
```